GETTING_STARTED.md

```diff
@@ -0,0 +1,448 @@
+  * [1.3. Editing / Publishing Separation](#13-editing--publishing-separation)
+* [2. Getting started](#2-getting-started)
+  * [2.1. Data retrieval](#21-data-retrieval)
+     * [Overview](#overview)
+     * [Retrieving tables](#retrieving-tables)
+     * [Retrieving columns](#retrieving-columns)
+     * [Retrieving rows](#retrieving-rows)
+     * [Retrieving a specific column](#retrieving-a-specific-column)
+     * [Retrieving a specific row](#retrieving-a-specific-row)
+     * [Retrieving a specific cell](#retrieving-a-specific-cell)
+  * [2.2. Data types &amp; column kinds](#22-data-types--column-kinds)
+     * [Primitive data types](#primitive-data-types)
+        * [Examples](#examples)
+     * [Complex data types](#complex-data-types)
+        * [<code>multi-language</code> and <code>multi-country</code>](#multi-language-and-multi-country)
+        * [<code>link</code>](#link)
+        * [<code>concat</code> and <code>group</code>](#concat-and-group)
+        * [<code>attachment</code>](#attachment)
+
+# 1. Preface
+
+GRUD is an acronym and stands for "generic relational enterprise database". The following document should give a short introduction to GRUD and its principles.
+
+## 1.1. Information is widely spread across the company
+In today's companies data and information is widely spread across divisions and departments. We saw the need for storing enterprise data in a structured and centralized way. A company could and should take advantage of connecting all this widespread data.
+
+The most valuable asset in your business is data. Managing data in a single source of truth is the crucial part of handling the complexity in today's rising demand for a digital Omnichannel strategy. GRUD enables you to spread content to multiple channels from one source. 
+
+## 1.2. Connecting the dots
+One of the main ideas is to store enterprise data in simple tables which are interconnected — which do have relations. You could start with simple and unconnected tables. If you start to connect the data in the right way, you will leverage from rich product information. This idea is not new, but every relational database (Oracle, PostgreSQL, MySQL, etc.) out there shows how powerful it can be. We took this idea and its simple core concepts and built a user & consumer centered enterprise database. The core parts of GRUD consists of an RESTful API and an easy-to-use web-based user interface.
+
+## 1.3. Editing / Publishing Separation
+Another driving force behind GRUD's software architecture is the principle "Separation of Concerns". Traditional content management systems ship both "content editing" & "content publishing" in one piece of software. Both systems are interleaved but still serve highly different use cases. Content editing and content publishing in an omnichannel universe do have totally different requirements. The life-cycle of e.g. a website is different to a central enterprise database like GRUD.
```

Comment:
different **from** a...

Reply:
Both seems to be correct but I will go for *from* as [recommended](https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/different-from-different-to-or-different-than) — my gut instinct told me otherwise.